ibu-imager/seedcreator/seedcreator.go

```diff
@@ -502,3 +511,35 @@ func (s *SeedCreator) renderInstallationEnvFile(recertContainerImage, seedFullDo
 
 	return utils.RenderTemplateFile(srcFile, substitutions, destFile, 0o644)
 }
+
+func (s *SeedCreator) deleteNode(ctx context.Context) error {
+	doneFile := "/var/tmp/node_deletion.done"
```

Comment:
This is not needed, the function is already idempotent without it

Reply:
Not fully, there can be a case where we already brought down kube-api and something else failed, example 
"delete all containers" and in this case this one will fail